expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -273,11 +273,11 @@ void shouldGetOrganizationIdsForAffiliations(Class<? extends TicketEntry> ticket
         throws ApiGatewayException {
         var publication = TicketTestUtils.createPersistedPublication(status, resourceService);
         var ticket = TicketTestUtils.createPersistedTicket(publication, ticketType, ticketService);
+        ticket.setOwnerAffiliation(randomUri());
 
-        var expectedOrgId = publication.getResourceOwner().getOwnerAffiliation();
         var orgIds = expansionService.getOrganization(ticket).id();
 
-        assertThat(orgIds, is(equalTo(expectedOrgId)));
```

Comment:
You should test for both cases of the "ticketEntry.getOwnerAffiliation() != null" statement.